WordPress/Classes/Utility/WPError+Swift.swift

```diff
@@ -0,0 +1,35 @@
+
+import Foundation
+import WordPressFlux
+
+extension WPError {
```

Comment:
Thanks for renaming this file, and refactoring the code here. I think it's turned out well 😀